eng/pipelines/libraries/build-job.yml

```diff
@@ -89,6 +88,13 @@ jobs:
               ./emsdk activate --embedded ${EMSCRIPTEN_VERSION}-upstream
             displayName: Install Emscripten
 
+          # Build CoreLib for WebAssembly
+          - script: $(_buildScript)
```

Comment:
Please pass in `-ci` parameter and override the binlog. `/bl:$(Build.SourcesDirectory)/artifacts/log/$(_BuildConfig)/BuildCorelib.binlog`

Reply:
this block is no longer required since webassembly now depends on mono